### ISSUE TITLE
Random volume rasing

### ISSUE THREAD

**Issue by emoon** [sasq64/chipmachine#64]
This is one of those "happens now and then" bug reports which such but I figured I would report it anyway.

After playing a bunch of music (random collection shuffle in my case) sometimes Chipmachine will raise the internal volume to really high so it will start clipping and everything sounds really bad. It doesn't seem to be related to a specific format or specific song (it happened to me on both YM and SID)

After switch to next song it goes away. It seems to happen more often when Chipmachine is looking for a new song to play (the "marker" on the top is shown) and can't find it for a while. This happens on macOS and version 1.3d but I don't know if it's OS related or not (I would guess it isn't)

Screenshot when it happens (notice the scope meters are really high and clipping)

![chipmachine](https://user-images.githubusercontent.com/350822/43761556-b7f04c34-9a25-11e8-86b6-8041f8fd1405.png)

**Comment by emoon** [comment]
Actually this seems to be happening all the time when a song has finished playing and chipmachine is looking for the next song to play (marker is shown)

**Comment by lmorchard** [comment]
Yeah, I've had this happen too. Almost like the fade out hits zero after the end of the song - but since the song keeps going while the next one loads, it's like the volume fade out crosses zero and starts incrementing instead

**Comment by sasq64** [comment]
That was a good tip, should be fixed by bcfafa20
